app/ts/ReportingUI/components/Reporting.tsx

```diff
@@ -150,39 +150,123 @@ type OutcomeStake = {
 	status: 'Winning' | 'Losing'
 	payoutNumerators: EthereumQuantity[]
 }
+
 interface DisplayStakesProps {
 	outcomeStakes: OptionalSignal<readonly OutcomeStake[]>
 	maybeAccountAddress: OptionalSignal<AccountAddress>
 	marketData: OptionalSignal<MarketData>
 }
+
 export const DisplayStakes = ({ outcomeStakes, maybeAccountAddress, marketData }: DisplayStakesProps) => {
 	if (outcomeStakes.deepValue === undefined) return <></>
 
-	const report = async (outcomeStake: OutcomeStake) => {
+	const selectedOutcome = useSignal<string | null>(null)
+	const reason = useSignal<string>('')
+	const amountInput = useSignal<string>('')
```

Comment:
I recommend having this be a `useOptionalSignal<bigint>()` and using that `Input` element I linked to previously.  That way you can be sure that this this will always be a properly parsed number, and not some unparsable string.

Reply:
yeah, I have issue about this. I haven't looked into it yet